modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,385 @@
+= Schema Registry Authorization
+:page-categories: Management, Schema Registry, rpk, Security
+:page-aliases: manage:schema-reg-authorization.adoc
+:description: Learn how to set up and manage Schema Registry Authorization using ACL definitions that control user access to specific Schema Registry operations.
+// tag::single-source[]
+
+Schema Registry Authorization enables fine-grained restriction of operations to Schema Registry resources by user or role through Access Control Lists (ACLs).
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+== About Schema Registry Authorization
+
+Schema Registry Authorization allows you to control which users and applications can perform specific operations within the Redpanda Schema Registry. This ensures that only authorized entities can read, write, modify, delete, or configure schemas and their settings.
+
+Before v25.2, Schema Registry supported authentication, but once a user was authenticated, they had full access to all Schema Registry operations, including reading, modifying, and deleting schemas and configuration both per-subject and globally.
+
+Starting in v25.2, Schema Registry Authorization provides fine-grained access control through ACLs. You can now restrict access to specific subjects and operations.
+
+=== How to manage Schema Registry Authorization
+
+You can manage Schema Registry Authorization in two ways:
+
+- **Using rpk**: Use the xref:reference:rpk/rpk-security/rpk-security-acl-create.adoc[`rpk security acl create`] command, just like you would for other Kafka ACLs.
+- **Using the API**: Use the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints.
+
+=== Schema Registry ACL resource types
+
+Schema Registry Authorization introduces two new ACL resource types in addition to the standard Kafka ACL resources (`topic`, `group`, `cluster`, and `transactionalID`):
+
+* `REGISTRY`: Controls whether or not to grant ACL access to global, or top-level Schema Registry operations. Specify using the flag `registry-global`.
+* `SUBJECT`: Controls ACL access for specific Schema Registry subjects. Specify using the flag `registry-subject`.
+
+== Supported operations
+
+Redpanda Schema Registry ACLs support the following specific subset of Schema Registry endpoints and operations:
+
+[NOTE]
+====
+Not all Kafka operations are supported when using Redpanda Schema Registry ACLs.
+====
+
+|===
+| Endpoint | HTTP Verb | Operation | Resource
+
+| `/config`
+| `GET`
+| `describe_configs`
+| `registry-global`
```

Comment:
@Feediver1 this may be accurate, but Im confused.  Above the resource is called 'REGISTRY' but below its called 'registry-global' . The latter may be in fact just the syntax we've chosen to use in RPK, but it doesn't map to the resources in the docs page describing ACLs in Redpanda overall, nor does it match the RPK readout. If that's the case, we might want to include the actual resource name (in all caps) that one would see when listing ACLS as another column.  One is the actual resource name and the other would be the RPK syntax.

Is this right @r-vasquez ?  cc @treevon

Reply:
Awaiting your response @r-vasquez @treevon